posthog/clickhouse/migrations/0019_group_analytics_materialized_columns.py

```diff
@@ -1,9 +1,12 @@
 from infi.clickhouse_orm import migrations
 
-from posthog.clickhouse.materialized_columns import materialize
-
 
 def create_materialized_columns(database):
+    try:
+        from ee.clickhouse.materialized_columns.columns import materialize
+    except ImportError:
+        return
```

Comment:
hey ted, probably need some context, but do we want this? I mean, if the materialize method can't be imported, we won't run the materialization, while before it's something that would be expected when this migration was run.

Not that it's going to have a huge impact since these migrations were already run, but just to know.

Reply:
Good call out. This one is a little tricky and part of the reason I wanted to do this cleanup was to make the behavior here more obvious since it wasn't always clear what was (or wasn't) happening for some of these calls.

The `materialize` that was previously here (`from posthog.clickhouse.materialized_columns import materialize`) was imported from https://github.com/PostHog/posthog/blob/7116d754dc7d1ebca1dd6ab414f714f77b661e9d/posthog/clickhouse/materialized_columns/__init__.py#L1-L6

So if the `ee.*` package was importable, things would previously work as they do here.

If the `ee.*` package couldn't be imported, it fell back to this version: https://github.com/PostHog/posthog/blob/7116d754dc7d1ebca1dd6ab414f714f77b661e9d/posthog/clickhouse/materialized_columns/column.py#L19-L26

So, this should work the same in that case as it did before (i.e. does nothing) but is hopefully a bit more obvious to future readers.

Reply:
oohhh fine then! thanks for the explanation!